### ISSUE TITLE
SyrinControl updated from stable to 0.4.0-beta2 without the consent

### ISSUE THREAD

**Issue by Majea** [frondeus/fvtt-syrin-control#71]
**Describe the bug**

Today, my syrinscape control plugin was automatically upgraded to 0.4.0-beta2. After this upgrade:
1. the syrinscape configuration no longer appears in the scene ambience tab
![image](https://user-images.githubusercontent.com/3779503/218266321-f2a4963a-670d-4608-8c19-10b81ce7c4e1.png)

2. the syrinscape controls no longer appear in the playlist section
![image](https://user-images.githubusercontent.com/3779503/218266353-1a7a258d-66f8-4312-91c5-635518d17837.png)

Please note I didn't manually install that beta release. It was part of global update of all my installed modules (global configuration and parameters of Foundry). 

**To Reproduce**

1. start Foundry
2. login into Foundry
3. update all your modules, assuming syrinscape control is installed
4. => this updates the syrinscape control plugin to 0.4.0-beta2
5. start a world with the syrinscape control plugin enabled
6. Open a scene
7. Right-click to open its configuration
8. click on the ambience tab
9. => the syrinscape configuration for the scene is not present anymore
10. close the scene configuration
11. open the playlists tab
12. => the syrinscape controls no longer appear in the playlist section. Only the syrinscape volume slider is still there. 
![image](https://user-images.githubusercontent.com/3779503/218266840-98a1d54d-b76e-480b-8cd3-afd0363f71f9.png)

**Expected behavior**

1. The syrinscape controls appear in the scene configuration, ambience tab
2. The syrinscape controls appear in the playlists tool

**Details**
OS: mac os ventura 13.1
Browser: chrome Version 109.0.5414.119 
Foundry: version 10 build 291

This was also tested on a fresh world with only socketlib (1.0.12) and syrinscape control installed.

**Logs**

```
_______________________________________________________________
 _____ ___  _   _ _   _ ____  ______   __ __     _______ _____ 
|  ___/ _ \| | | | \ | |  _ \|  _ \ \ / / \ \   / |_   _|_   _|
| |_ | | | | | | |  \| | | | | |_) \ V /   \ \ / /  | |   | |  
|  _|| |_| | |_| | |\  | |_| |  _ < | |     \ V /   | |   | |  
|_|   \___/ \___/|_| \_|____/|_| \_\|_|      \_/    |_|   |_|  
===============================================================
foundry.js:79688 Foundry VTT | Reestablishing existing session 91d291d811abba38506e7840
foundry.js:7711 Foundry VTT | Connected to server socket using session 91d291d811abba38506e7840
foundry.js:7807 Foundry VTT | Initializing Foundry Virtual Tabletop Game
dnd5e.mjs? [sm]:46 DnD5e | Initializing the DnD5e Game System - Version 2.1.4
_______________________________
______      ______ _____ _____
|  _  \___  |  _  \  ___|  ___|
| | | ( _ ) | | | |___ \| |__
| | | / _ \/\ | | |   \ \  __|
| |/ / (_>  < |/ //\__/ / |___
|___/ \___/\/___/ \____/\____/
_______________________________
main.ts:17 SyrinControl | Initializing...
foundry.js:3133 Foundry VTT | Synchronized official game time in 11ms
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/active-effects.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/apps/parts/trait-list.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-traits.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-inventory.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-features.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-spellbook.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-warnings.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-action.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-activation.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-advancement.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-description.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-mountable.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-spellcasting.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-summary.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/journal/parts/journal-table.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/advancement/parts/advancement-controls.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/advancement/parts/advancement-spell-config.hbs
foundry.js:6771 Foundry VTT | Loaded localization file modules/fr-core/fr.json
foundry.js:6771 Foundry VTT | Loaded localization file modules/fvtt-syrin-control/languages/en.json
foundry.js:6771 Foundry VTT | Loaded localization file lang/en.json
foundry.js:6771 Foundry VTT | Loaded localization file systems/dnd5e/lang/en.json
utils.ts:39 SyrinControl | SocketLib registered | Player
foundry.js:5099 Foundry VTT | Rendering SceneNavigation
foundry.js:5099 Foundry VTT | Rendering Notifications
foundry.js:5099 Foundry VTT | Rendering Sidebar
foundry.js:5099 Foundry VTT | Rendering PlayerList
foundry.js:5099 Foundry VTT | Rendering Hotbar
foundry.js:5099 Foundry VTT | Rendering Pause
foundry.js:5099 Foundry VTT | Rendering SceneControls
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/navigation.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/sidebar.html
foundry.js:5099 Foundry VTT | Rendering ActorDirectory
foundry.js:5099 Foundry VTT | Rendering CardsDirectory
foundry.js:5099 Foundry VTT | Rendering ChatLog
foundry.js:5099 Foundry VTT | Rendering CombatTracker5e
foundry.js:5099 Foundry VTT | Rendering CompendiumDirectory
foundry.js:5099 Foundry VTT | Rendering ItemDirectory
foundry.js:5099 Foundry VTT | Rendering JournalDirectory
foundry.js:5099 Foundry VTT | Rendering PlaylistDirectory
foundry.js:5099 Foundry VTT | Rendering SceneDirectory
foundry.js:5099 Foundry VTT | Rendering Settings
foundry.js:5099 Foundry VTT | Rendering RollTableDirectory
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/user/players.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/hotbar.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/pause.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/controls.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/chat-log.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/combat-tracker.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/compendium-directory.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/scene-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/settings.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/playlist-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
2foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-directory.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/chat-message.html
4foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-directory.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/playlists-directory.html
foundry.js:63231 Foundry VTT | Fonts loaded and ready.
utils.min.mjs:8 
    PixiJS 6.5.2 - ✰ WebGL 2 ✰      http://www.pixijs.com/    ♥♥♥ 
foundry.js:20900 Foundry VTT | Viewing Scene test
foundry.js:23648 Foundry VTT | Drawing game canvas for scene test
foundry.js:25946 Foundry VTT | Loaded icons/svg/combat.svg (2%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/cowled.svg (4%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/aura.svg (6%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/padlock.svg (9%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/up.svg (11%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/down.svg (13%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/light.svg (15%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/skull.svg (17%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sound-off.svg (19%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/explosion.svg (21%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/light-off.svg (23%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sound.svg (26%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-open-outline.svg (28%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-locked-outline.svg (30%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sleep.svg (32%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-secret-outline.svg (34%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/unconscious.svg (36%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-closed-outline.svg (38%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/daze.svg (40%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/wing.svg (43%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/blind.svg (45%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/falling.svg (47%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/net.svg (49%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/paralysis.svg (51%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/deaf.svg (53%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/fire.svg (55%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/silenced.svg (57%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/lightning.svg (60%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/terror.svg (62%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/frozen.svg (64%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/acid.svg (66%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/poison.svg (68%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/biohazard.svg (70%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/blood.svg (72%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/regen.svg (74%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sun.svg (77%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/degen.svg (79%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/invisible.svg (81%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/target.svg (83%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/upgrade.svg (85%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/eye.svg (87%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/downgrade.svg (89%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/angel.svg (91%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/ice-shield.svg (94%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/fire-shield.svg (96%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/holy-shield.svg (98%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/mage-shield.svg (100%)
foundry.js:30347 Foundry VTT | Drawing the WeatherEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasBackgroundAlterationEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasIlluminationEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasColorationEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasVisibility canvas layer
foundry.js:30347 Foundry VTT | Drawing the GridLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the DrawingsLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the TemplateLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the TilesLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the WallsLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the TokenLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the SoundsLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the LightingLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the NotesLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the ControlsLayer canvas layer
foundry.js:5099 Foundry VTT | Rendering HeadsUpDisplay
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/hud.html
utils.ts:39 SyrinControl | Ready...
utils.ts:32 SyrinControl | Subscribe | next mood:  undefined
trace @ utils.ts:32
(anonymous) @ main.ts:123
(anonymous) @ stores.ts:312
subscribe @ index.mjs:50
subscribe @ stores.ts:304
(anonymous) @ main.ts:122
await in (anonymous) (async)
#call @ foundry.js:724
callAll @ foundry.js:681
setupGame @ foundry.js:7952
commons.js:6946 Foundry VTT | Created Setting with id [QUX4Jt1uMl0k2kJ2]
```

**Comment by Majea** [comment]
**To Reproduce**

<!-- Steps to reproduce the behavior:
1. start Foundry
2. login into Foundry
3. update all your modules, assuming syrinscape control is installed
4. => this updates the syrinscape control plugin to 0.4.0-beta2
5. start a world with the syrinscape control plugin enabled
6. Open a scene
7. Right-click to open its configuration
8. click on the ambience tab
9. => the syrinscape configuration for the scene is not present anymore
10. close the scene configuration
11. open the playlists tab
12. => the syrinscape controls no longer appear in the playlist section. Only the syrinscape volume slider is still there. 
![image](https://user-images.githubusercontent.com/3779503/218266840-98a1d54d-b76e-480b-8cd3-afd0363f71f9.png)


**Expected behavior**

1. The syrinscape controls appear in the scene configuration, ambience tab
2. The syrinscape controls appear in the playlists tool

**Details**
OS: mac os ventura 13.1
Browser: chrome Version 109.0.5414.119 
Foundry: version 10 build 291

This was also tested on a fresh world with only socketlib (1.0.12) and syrinscape control installed.

**Logs**

```
_______________________________________________________________
 _____ ___  _   _ _   _ ____  ______   __ __     _______ _____ 
|  ___/ _ \| | | | \ | |  _ \|  _ \ \ / / \ \   / |_   _|_   _|
| |_ | | | | | | |  \| | | | | |_) \ V /   \ \ / /  | |   | |  
|  _|| |_| | |_| | |\  | |_| |  _ < | |     \ V /   | |   | |  
|_|   \___/ \___/|_| \_|____/|_| \_\|_|      \_/    |_|   |_|  
===============================================================
foundry.js:79688 Foundry VTT | Reestablishing existing session 91d291d811abba38506e7840
foundry.js:7711 Foundry VTT | Connected to server socket using session 91d291d811abba38506e7840
foundry.js:7807 Foundry VTT | Initializing Foundry Virtual Tabletop Game
dnd5e.mjs? [sm]:46 DnD5e | Initializing the DnD5e Game System - Version 2.1.4
_______________________________
______      ______ _____ _____
|  _  \___  |  _  \  ___|  ___|
| | | ( _ ) | | | |___ \| |__
| | | / _ \/\ | | |   \ \  __|
| |/ / (_>  < |/ //\__/ / |___
|___/ \___/\/___/ \____/\____/
_______________________________
main.ts:17 SyrinControl | Initializing...
foundry.js:3133 Foundry VTT | Synchronized official game time in 11ms
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/active-effects.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/apps/parts/trait-list.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-traits.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-inventory.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-features.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-spellbook.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/parts/actor-warnings.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-action.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-activation.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-advancement.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-description.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-mountable.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-spellcasting.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/items/parts/item-summary.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/journal/parts/journal-table.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/advancement/parts/advancement-controls.hbs
2foundry.js:6887 Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/advancement/parts/advancement-spell-config.hbs
foundry.js:6771 Foundry VTT | Loaded localization file modules/fr-core/fr.json
foundry.js:6771 Foundry VTT | Loaded localization file modules/fvtt-syrin-control/languages/en.json
foundry.js:6771 Foundry VTT | Loaded localization file lang/en.json
foundry.js:6771 Foundry VTT | Loaded localization file systems/dnd5e/lang/en.json
utils.ts:39 SyrinControl | SocketLib registered | Player
foundry.js:5099 Foundry VTT | Rendering SceneNavigation
foundry.js:5099 Foundry VTT | Rendering Notifications
foundry.js:5099 Foundry VTT | Rendering Sidebar
foundry.js:5099 Foundry VTT | Rendering PlayerList
foundry.js:5099 Foundry VTT | Rendering Hotbar
foundry.js:5099 Foundry VTT | Rendering Pause
foundry.js:5099 Foundry VTT | Rendering SceneControls
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/navigation.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/sidebar.html
foundry.js:5099 Foundry VTT | Rendering ActorDirectory
foundry.js:5099 Foundry VTT | Rendering CardsDirectory
foundry.js:5099 Foundry VTT | Rendering ChatLog
foundry.js:5099 Foundry VTT | Rendering CombatTracker5e
foundry.js:5099 Foundry VTT | Rendering CompendiumDirectory
foundry.js:5099 Foundry VTT | Rendering ItemDirectory
foundry.js:5099 Foundry VTT | Rendering JournalDirectory
foundry.js:5099 Foundry VTT | Rendering PlaylistDirectory
foundry.js:5099 Foundry VTT | Rendering SceneDirectory
foundry.js:5099 Foundry VTT | Rendering Settings
foundry.js:5099 Foundry VTT | Rendering RollTableDirectory
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/user/players.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/hotbar.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/pause.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/controls.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/chat-log.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/combat-tracker.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/compendium-directory.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/scene-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/settings.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/playlist-partial.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/folder-partial.html
2foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-directory.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/chat-message.html
4foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/document-directory.html
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/sidebar/playlists-directory.html
foundry.js:63231 Foundry VTT | Fonts loaded and ready.
utils.min.mjs:8 
    PixiJS 6.5.2 - ✰ WebGL 2 ✰      http://www.pixijs.com/    ♥♥♥ 
foundry.js:20900 Foundry VTT | Viewing Scene test
foundry.js:23648 Foundry VTT | Drawing game canvas for scene test
foundry.js:25946 Foundry VTT | Loaded icons/svg/combat.svg (2%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/cowled.svg (4%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/aura.svg (6%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/padlock.svg (9%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/up.svg (11%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/down.svg (13%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/light.svg (15%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/skull.svg (17%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sound-off.svg (19%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/explosion.svg (21%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/light-off.svg (23%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sound.svg (26%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-open-outline.svg (28%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-locked-outline.svg (30%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sleep.svg (32%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-secret-outline.svg (34%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/unconscious.svg (36%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/door-closed-outline.svg (38%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/daze.svg (40%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/wing.svg (43%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/blind.svg (45%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/falling.svg (47%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/net.svg (49%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/paralysis.svg (51%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/deaf.svg (53%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/fire.svg (55%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/silenced.svg (57%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/lightning.svg (60%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/terror.svg (62%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/frozen.svg (64%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/acid.svg (66%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/poison.svg (68%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/biohazard.svg (70%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/blood.svg (72%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/regen.svg (74%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/sun.svg (77%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/degen.svg (79%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/invisible.svg (81%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/target.svg (83%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/upgrade.svg (85%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/eye.svg (87%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/downgrade.svg (89%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/angel.svg (91%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/ice-shield.svg (94%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/fire-shield.svg (96%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/holy-shield.svg (98%)
foundry.js:25946 Foundry VTT | Loaded icons/svg/mage-shield.svg (100%)
foundry.js:30347 Foundry VTT | Drawing the WeatherEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasBackgroundAlterationEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasIlluminationEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasColorationEffects canvas layer
foundry.js:30347 Foundry VTT | Drawing the CanvasVisibility canvas layer
foundry.js:30347 Foundry VTT | Drawing the GridLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the DrawingsLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the TemplateLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the TilesLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the WallsLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the TokenLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the SoundsLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the LightingLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the NotesLayer canvas layer
foundry.js:30347 Foundry VTT | Drawing the ControlsLayer canvas layer
foundry.js:5099 Foundry VTT | Rendering HeadsUpDisplay
foundry.js:6887 Foundry VTT | Retrieved and compiled template templates/hud/hud.html
utils.ts:39 SyrinControl | Ready...
utils.ts:32 SyrinControl | Subscribe | next mood:  undefined
trace @ utils.ts:32
(anonymous) @ main.ts:123
(anonymous) @ stores.ts:312
subscribe @ index.mjs:50
subscribe @ stores.ts:304
(anonymous) @ main.ts:122
await in (anonymous) (async)
#call @ foundry.js:724
callAll @ foundry.js:681
setupGame @ foundry.js:7952
commons.js:6946 Foundry VTT | Created Setting with id [QUX4Jt1uMl0k2kJ2]
```

**Comment by Majea** [comment]
Note: if you uninstall the syrinscape control plugin and install it again, it goes back to 0.3.1, which doesn't have this bug. However, as soon as you hit the update button in the global plugins page, it updates to 0.4.0-beta2. Hence a workaround is to reinstall the plugin after each global update of your plugins. This brings us back to 0.3.1.

**Comment by frondeus** [comment]
Uhm, I am trying to understand why it updated automatically to 0.4.0-beta2 since... I haven't published it yet via the official FoundryVTT admin panel! 

I always mark my releases as pre-release and do not push changes to the admin panel to avoid auto-update and confusion. I only publish the alpha/betas via GitHub so you have to manually paste the link to the manifest.

Note - there was no official discord message saying that the new SyrinControl version was released in the FoundryVTT server.

Therefore I am very confused about how it happened.

Anyway - if you decide to keep the beta version - there is also 0.4.0-beta5 that has some bug fixes.

AD. 1.  This is not a bug but a breaking change specified in the changelog:
https://github.com/frondeus/fvtt-syrin-control/releases/tag/0.4.0-alpha-3
> Since there is playlist support, scene configuration no longer contains an option to set the mood. Instead, I recommend importing the selected mood and setting it with the native ambiance setting. It just works :)

Therefore, you can import your favorite moods/soundsets as playlists and then use native integration between playlists and scenes. 

AD 2. Again, if you address the lack of mood/soundset selection dropdowns - that's actually the feature - instead, you have a new button, "Import Soundsets", where is the search text box as well as a list of your soundsets and moods. You can trigger moods directly from that dialog or use the import feature and have it as a "native" playlist.

If you mean the lack of volume sliders - those are now placed with other volumes.

**Comment by Majea** [comment]
Ok, all this makes sense. I hadn't realised that this "Import Soundsets" was from the Syrinscape plugin.

**Comment by frondeus** [comment]
I'm gonna keep it open, although I will change the title so I know where the root problem lies :)

**Comment by frondeus** [comment]
I released the 0.4, stable version and that should no longer be an issue.
Anyway, I know what I did wrong so I wont repeat this mistake :)
